include/maliput_sparse/builder/builder.h

```diff
@@ -0,0 +1,360 @@
+
+  /// @brief Set the right maliput_sparse::geometry::LineString of the LaneGeometry.
+  /// @param left_line_string The right maliput_sparse::geometry::LineString to set in the LaneGeometry.
+  /// @return A reference to this LaneGeometryBuilder.
+  LaneGeometryBuilder& RightLineString(const maliput_sparse::geometry::LineString3d& right_line_string);
+
+  /// @brief Finalizes the construction of the LaneGeometry and sets it to the parent LaneBuilder.
+  /// @pre Left and right LineStrings must be set before calling this method.
+  /// @throws maliput::common::assertion_error When the left and right LineStrings were not set.
+  /// @return The reference to the parent LaneBuilder.
+  LaneBuilder& EndLaneGeometry();
+
+ private:
+  std::optional<maliput_sparse::geometry::LineString3d> left_line_string_{};
+  std::optional<maliput_sparse::geometry::LineString3d> right_line_string_{};
+};
+
+/// @brief Builder class for maliput::api::Lanes.
+class LaneBuilder final : public details::NestedBuilder<SegmentBuilder> {
+ public:
+  /// @brief Construct a new Lane Builder object.
+  /// @param parent The parent SegmentBuilder. It must not be nullptr.
+  explicit LaneBuilder(SegmentBuilder* parent) : details::NestedBuilder<SegmentBuilder>(parent) {}
+
+  /// @brief Sets the maliput::api::LaneId of the maliput::api::Lane.
+  /// @param lane_id The maliput::api::LaneId.
+  /// @return A reference to this LaneBuilder.
+  LaneBuilder& Id(const maliput::api::LaneId& lane_id);
+
+  /// @brief Sets the maliput::api::maliput::api::HBounds of the maliput::api::Lane.
+  /// @param hbounds A maliput::api::HBounds to set to the Lane.
+  /// @return A reference to this LaneBuilder.
+  LaneBuilder& HeightBounds(const maliput::api::HBounds& hbounds);
+
+  /// @brief Starts the LaneGeometry builder for this Lane.
+  /// @return A LaneGeometryBuilder.
+  LaneGeometryBuilder StartLaneGeometry();
+
+  /// @brief Finalizes the construction process of this Lane by inserting the Lane into the
+  /// parent SegmentBuilder.
+  /// @throws maliput::common::assertion_error When there is no LaneGeometry to be set into the Lane.
+  /// @return A reference to the SegmentBuilder.
+  SegmentBuilder& EndLane();
+
+  /// @brief Sets a maliput_sparse::geometry::LaneGeometry into this builder to fill in the Lane.
+  /// @details This method is only intended to be called by LaneGeometryBuilder instances.
+  /// @see maliput::common::Passkey class description for further details.
+  /// @param lane_geometry A maliput_sparse::geometry::LaneGeometry to be stored into the Lane. It must not be nullptr.
+  /// @throws maliput::common::assertion_error When @p lane_geometry is nullptr.
+  void SetLaneGeometry(maliput::common::Passkey<LaneGeometryBuilder>,
+                       std::unique_ptr<maliput_sparse::geometry::LaneGeometry> lane_geometry);
+
+ private:
+  maliput::api::LaneId id_{"unset_id"};
+  maliput::api::HBounds hbounds_{};
```

Comment:
Probably we should add a default HBounds here. Like `{0., 5.}`

Reply:
Done. I also changed some test cases to check the default value.